sympy/matrices/determinant.py

```diff
@@ -380,39 +385,13 @@ def _charpoly(M, x='lambda', simplify=_simplify):
     >>> M.charpoly(x).gen == x
     False
 
-    Notes
-    =====
-
-    The Samuelson-Berkowitz algorithm is used to compute
-    the characteristic polynomial efficiently and without any
-    division operations.  Thus the characteristic polynomial over any
-    commutative ring without zero divisors can be computed.
```

Comment:
This is still using the Berkowitz algorithm. It's just using it on `DDM` rather than `Matrix`

Reply:
Surely will add it.
Thanks for informing.